.github/workflows/ci.yml

```diff
@@ -7,7 +7,13 @@ on:
       - main
       - stable/*
       - release-*
-  pull_request: {}
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - labeled
+      - unlabeled
```

Comment:
:question: What's the purpose behind that change?

Reply:
The default types for PR are `opened`, `synchronize`, and `reopened`. So I added these to keep the behavior as before. The idea is we now need `labeled` and `unlabeled`, which are not triggers, if we want to also trigger CI to run again when we add/remove the bypass label.

The downside is that now every time you label your PR, the CI will run...I will look with Christian how we can find a better middle ground.

Reply:
In the end, we will go with `BREAKING CHANGE:` must be in the last commit (for merge_group and push), and in the PR body for a PR. The PR body is then the message of the merge commit, so it will be the last commit when merging/pushing.